### ISSUE TITLE
Scheduler

### ISSUE THREAD

**Issue by leminhhuy132** [Lornatang/SRGAN-PyTorch#53]
I don't understand why using define optimizer in line 51 train_srgan.py. Can you help me, thank you!

**Comment by Lornatang** [comment]
According to the definition of the adjustment method for the learning rate in the paper

**Comment by leminhhuy132** [comment]
I mean why "d_scheduler, g_scheduler = define_optimizer(discriminator, generator)" doesn't use define_scheduler function.

**Comment by Lornatang** [comment]
Thanks for pointing this out, I wrote it wrong

**Comment by Lornatang** [comment]
This low-level bug has now been fixed....
